### ISSUE TITLE
running the doc server and getting 404 no matter what

### ISSUE THREAD

**Issue by costezki** [proycon/foliadocserve#15]
I would appreciate a hand running FLAT and correspondingly the foliadocserve server. 
Currently no matter what I get 404s on every interface action, and if I write a query, any FQL query, I get 500s. 

Of course flat could not run because the doc server is not started properly. Please let me know what am I missing. 

I take the following actions:

1. Run the server
``` 
foliadocserve -d ./doc_repo -p 8080 --git -l ./foliadocserver.log
```
2. accessing the web interface
```
http://localhost:8080/
```
3. click on list namespaces and get 404 response
4. execute 'USE namespace' FQLquery and receive 500 response

The Stack trace is provided the files below:

[stac-trace-folia.log](https://github.com/proycon/foliadocserve/files/4724841/stac-trace-folia.log)

[foliadocserver.log](https://github.com/proycon/foliadocserve/files/4724839/foliadocserver.log)

**Comment by proycon** [comment]
I assume there is nothing in the ``doc_repo`` directory yet? That's why you are getting 404 errors, but the document server does seem to run correctly.  I admit the foliadocserve interface is a bit simple and understand the confusion, most people won't be using it directly.

I'm more wondering why FLAT won't work in your case, the document server is running so you should be able to upload FoLiA documents using FLAT (the upload in the foliadocserve web interface isn't properly implemented yet)

You can start foliadocserve with something like  ``-D 5`` to set a higher debug level and get more information in the log, that may provide a clue if FLAT has trouble communicating with it.

**Comment by costezki** [comment]
I created in the ./doc_repo a subfolder _ns_ and copied _frog-deep-upgraded.2.0.2.folia.xml_ there.

When I run the Folia server I get a 404, as well, related to the namespace. 

The settingts.py and the bash files for running each system are attached here. 
[settings.zip](https://github.com/proycon/foliadocserve/files/4728918/settings.zip)

[folia-scripts.zip](https://github.com/proycon/foliadocserve/files/4728923/folia-scripts.zip)

![image](https://user-images.githubusercontent.com/13567133/83739564-0cc72f00-a656-11ea-8815-31ec0cab030b.png)
![image](https://user-images.githubusercontent.com/13567133/83739889-72b3b680-a656-11ea-8bb7-e951b3d5564f.png)

**Comment by proycon** [comment]
I've replicated your directory structure and I indeed get similar errors, I'm investigating whether there's a bug that crept in, as this is indeed not the behaviour I'd expect. Your scripts and all seem fine too.

**Comment by proycon** [comment]
Okay, I see the problem. Things go wrong because the document root directory that was supplied is not an absolute path, if you turn it into an absolute path then things do work. Since this is obviously not very intuitive I'll adapt foliadocserve to do some extra checks on startup and attempt to convert any relative paths to absolute ones automatically.

**Comment by proycon** [comment]
This should now be fixed in v0.7.3 (just released)
